Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN go build -o ../../tmp/_output/bin/move-blobs
 FROM registry.ci.openshift.org/ocp/4.16:base-rhel9
 COPY images/bin/entrypoint.sh /usr/bin/
 COPY manifests/image-references manifests/0* /manifests/
-COPY vendor/github.com/openshift/api/imageregistry/v1/*.crd.yaml /manifests/
+COPY vendor/github.com/openshift/api/imageregistry/v1/**/*.crd.yaml /manifests/
```

Comment:
The manifests were created due to featuregate in a nested directory. Above change includes them to avoid test failures for missing crds